src/models/elan1-r1/README.md

```diff
@@ -0,0 +1,29 @@
+# Eland 1U (elan1-r1)
+
+![Eland 1U](./img/elan1-r1-frontports.avif)
+
+The System76 Eland 1U is a rack-mount server with the following specifications:
+
+- Operating System
+    - Ubuntu Server 22.04 LTS (64-bit)
```

Comment:
We discussed adding shipping OS versions to product pages and I was generally in favor of it (we still have https://github.com/system76/tech-docs/issues/123 open for this), but Jeremy didn't like the idea, IIRC. Is this something we want to commit to right now?

Reply:
I think it makes more sense here as we *don't* ship with Pop!OS on these systems...and are more likely to only ship a LTS base.

Reply:
Makes sense to just remove it, IMO. It's kind of a given that some variety of Ubuntu Server shipped with it, and older LTS's would probably run fine on it as well.